packages/performance-tests/test/cases/prepare-fork.ts

```diff
@@ -42,6 +42,56 @@ export default async function prepareFork(sourceDb: BriefcaseDb, addReport: (...
       entityProcessingTimer?.elapsedSeconds ?? -1,
     );
   }
+
+  const targetPath1 = initOutputFile(`RawFork-${sourceDb.name}-target.bim`, outputDir);
```

Comment:
can you rename from targetPath1 to `rawForkTargetPath`?

Reply:
done